pressbooks/pressbooks
# Add anchors to interactive elements in export routines

Issue: https://github.com/pressbooks/pressbooks/issues/2287 - Items 3.

This change adds the link to the interactive elements (audio & videos) in the chapter. Currently we don't specify with anchors the specific link to the interactive elements in PDF, XHTML and HTMLBook exports.

### Test
- Pick a book and in a chapter ready to export, add multiple audios and/or videos.
- Export the book to PDF, XHTML and HTMLBook formats.
- Make sure the templates for audios and videos are rendered accordingly with the link to the interactive element.

Examples about what to expect in terms of output:
![Screenshot from 2021-11-11 08-40-43](https://user-images.githubusercontent.com/13248424/141308093-72429914-b2b1-48e5-b1ed-4059c7097c6c.png)
![Screenshot from 2021-11-11 08-40-38](https://user-images.githubusercontent.com/13248424/141308096-85c97ea5-cf79-48f8-8412-12208b7d354a.png)
